lib/challenge.js

```diff
@@ -52,6 +70,15 @@ var some_pet;
 
 // Answer Starts Here
 
+var some_pet = {
+
+  name: "Bailey Boxer",
+  species: "Boxer",
+  breed: "mixed",
+  noise: "Grrrr"
+  ],
+},
```

Comment:
Aside from that loose square  and unnecessary comma bracket above, this is a good object literal. The question asks you to put a `makeNoise` method on it, though. Half credit.